src/stylesheets/screens/question.scss

```diff
@@ -0,0 +1,69 @@
+.questions {
+  width: 100vw;
+  height: 100vh;
+
+  &__header {
+    @include header-container();
+    height: rem(60px);
+
+    padding: rem(16px 16px 0);
+  }
+
+  &__header-right {
+    order: 2;
+    margin-left: auto;
+  }
+
+  &__container {
+    @include details-container();
+    position: relative;
+
+    flex: 1 1 auto;
+    height: calc(100vh - 135px);
+  }
+
+  &__details {
+    margin: auto 0;
+  }
+
+  &__number {
+    color: map-get($brand-white, 'primary');
+    font-size: map_get($font-sizes, 'sm');
```

Comment:
I just checked the `$font-sizes` on `_variables.scss` but the `sm` size is not the same as on the design 🤔  the current one is `13px` but this should be `15px` so I think we can either adjust the variable but it might affect other typos as well or just use the `md` here instead 

![Screen Shot 2563-11-11 at 17 17 33](https://user-images.githubusercontent.com/1772999/98799851-3c1e6b00-2442-11eb-8c14-3409c7a2cda5.png)
![Screen Shot 2563-11-11 at 17 18 42](https://user-images.githubusercontent.com/1772999/98799857-3e80c500-2442-11eb-8b6f-bf1a782ceded.png)

for convenience, you can also add some comment on the `$font-sizes` variable so it's easier to know the pixel value of each font sizes 🙌 

```
$font-sizes: (
  sm: 0.8125rem,  // 13px
  md: 0.9375rem,  // 15px
  lg: 1.0625rem,  // 17px
  xl: 1.375rem,   // 22px
  xxl: 1.75rem,   // 28px
  xxxl: 2.125rem, // 34px
  xxxxl: 4rem     // 64px
);
```

Reply:
Changed the font size. THanks for pointing it out. I will fix it in a different PR soon.